app/src/main/res/layout/image_card_view.xml

```diff
@@ -159,7 +168,7 @@
                 android:layout_alignParentStart="true"
                 android:layout_marginStart="@dimen/lb_basic_card_info_text_margin"
                 android:fontFamily="sans-serif-condensed"
-                android:textColor="@color/lb_basic_card_title_text_color"
+                android:textColor="@color/gray_gradient_end"
```

Comment:
Can you make a new color for this? Because themes might want to color it in the future.